### ISSUE TITLE
Controlnet Finetune

### ISSUE THREAD

**Issue by VLadImirluren** [snap-research/ac3d#8]
Solid work!

In ac3d, you said using controlnet only fine-tune on the previous 8 layers of VDiT and trained with RFlow.

I checked the CogVideoX and found that the scheduler it used followed LDM setting (not RFlow).

I am curious about whether is it enough for fine-tuning diffusion to another objective using only controlnet?

Thanks

**Comment by VLadImirluren** [comment]
Thanks for your reply.

But my curiosity still exists. Is it enough to change the objective only training controlnet?

**Comment by VLadImirluren** [comment]
> The paper refers to the SnapVideo video DiT model. This is just a reimplementation of the paper on top of CogVideoX. It doesn't make a difference if the base model is built upon rectified flow or any other framework.

Thanks for your reply.

But my curiosity still exists. Is it enough to change the objective only training controlnet?

**Comment by sherwinbahmani** [comment]
I don't understand the question, you want to change the training objective of CogVideoX to rectified flow?

**Comment by VLadImirluren** [comment]
> I don't understand the question, you want to change the training objective of CogVideoX to rectified flow?

Yeah. Is it possible?

**Comment by sherwinbahmani** [comment]
I don't know, I didn't try it and I don't understand why it should be changed. Also it's not related to this work? Better to ask in the CogVideoX github?

**Comment by VLadImirluren** [comment]
> I don't know, I didn't try it and I don't understand why it should be changed. Also it's not related to this work? Better to ask in the CogVideoX github?

Yeah. Thanks to your patient reply. Apologies for my inappropriate question.
